packages/web-components/package.json

```diff
@@ -72,10 +72,10 @@
     "mocha": "7.2.0"
   },
   "dependencies": {
-    "@microsoft/fast-colors": "^5.3.0",
-    "@microsoft/fast-element": "^1.11.0",
-    "@microsoft/fast-foundation": "^2.47.0",
-    "@microsoft/fast-web-utilities": "^5.4.0",
+    "@microsoft/fast-element": "^2.0.0-beta.17",
+    "@microsoft/fast-foundation": "^3.0.0-alpha.21",
+    "@microsoft/fast-web-utilities": "^6.0.0",
+    "@fluentui/react-theme": "^9.1.1",
```

Comment:
This adds dependency on a `react-theme` package 💣
There is nothing React related in that package besides two things:
1. It has **react** in the package name.
2. It adds react as a peer dependency.

If we decide to go this way, we should do one of the following things:
- Rename `react-theme` to `theme`, get rid of the peer dependency (might be breaking change for FUIRv9).
- Hoist the tokens from `react-theme` to `theme`, re-export in `react-theme`.
- Use token pipeline to generate the same theme files for `web-components`.

Reply:
ideally we should use same fluent tokens and themes in all web experiences, thus

if `react-theme` doesn't contain anything react specific besides the peerDep

- deprecate `react-theme` on npm
- rename `react-theme` to `react`  (removal of peer Deps )
  - having react-theme with re-exports of theme adds unnecessary maintenance  

> might be breaking change for FUIRv9

from react-components api POV there will be no breaking change. once someone installs new feature bump of react-components they will get `theme` instead of `react-theme` in they node_modules

Reply:
If there is no use of React in `react-theme` why can't we just create a `theme` package and use that directly in `react-components`?

Reply:
That's what @Hotell is proposing (+ npm deprecation). I was not sure whether we would consider it a breaking change or not.

Reply:
Agree that the change is not breaking.

Reply:
Was just chatting with @chrisdholt today and we'd definitely love to have a shared, library-agnostic theme package that we can base both React and Web Components on. Sounds like folks here arrived at the same conclusion. +1 from me.

Reply:
+1 lets start from here